### DISCUSSION TITLE
New Labels – Share your opinions here

### DISCUSSION THREAD

**Discussion by leahcimto** [theforumhelpers/theforumhelpers.github.io#151] [General]
Hey! We've had the new labels for about 5 days now and I would like to hear your feedback on them! Do you like them, do you think they make things look cluttered, or do you think something else? Share here.

If the majority of users here don't like the new system I will get rid of it.

Tagging users that have been contributing:
@Accio1 @Shluffy @FunctionalMetatable @gosoccerboy5 @CodeGuy92

**Comment by FunctionalMetatable** [comment]
it works so yeah I guess its good

**Comment by Shluffy** [comment]
It is quite good. the bot on #152 was quite funny.

**Comment by aidan-crockett** [comment]
Oof, sorry for not noticing (I should check my email more lol)
love it! the tags are very clear

**Comment by leahcimto** [comment]
That's great! @Accio1 do you have any thoughts?

**Reply by Accio1** [reply]
They are less cluttered than I thought they would be, and I think it is good for organization. One thing that I am concerned about though is maybe not being able to use the actions labeller because the names contain colons. I'm still looking into if there is a way to have colons in label names while using actions labeller.

**Comment by leahcimto** [comment]
Since everyone seems to be in favor of these new labels they will be kept.
